Makefile.work

```diff
@@ -225,7 +225,11 @@ SLAVE_TAG = $(shell \
 COLLECT_DOCKER=DEFAULT_CONTAINER_REGISTRY=$(DEFAULT_CONTAINER_REGISTRY) \
 	scripts/collect_docker_version_files.sh \
 	$(SLAVE_BASE_IMAGE):$(SLAVE_BASE_TAG) \
-		target 
+		target \
+				$(SLAVE_BASE_IMAGE):$(SLAVE_BASE_TAG) \
```

Comment:
This line is the same as 227.
When they will be different?

Reply:
For the sonic-slave image, both are the same, but for the sonic dockers, the first one takes the docker name and the second one takes the docker image tag.
Eg:
+ DOCKER_IMAGE=docker-database
+ TARGET_PATH=target
+ DOCKER_IMAGE_TAG=docker-database-username:49592b425b7

Reply:
Suggest adding parameters when it is used.
It is confusing in this PR.

Reply:
Got it, will change this.

Reply:
done